### ISSUE TITLE
Namespaces include the node name where they previously appear to not do so

### ISSUE THREAD

**Issue by wjwwood** [ros-visualization/rqt_graph#6]
While reviewing this tutorial for ROS Lunar:

http://wiki.ros.org/ROS/Tutorials/UnderstandingTopics

I noticed that the current `rqt_graph` rendering looks different than the historical screenshots in the wiki page. For example:

![screenshot from 2017-05-11 15-20-38](https://cloud.githubusercontent.com/assets/100427/25974340/6c80939e-365d-11e7-933c-a483d6b2b969.png)

If I uncheck the namespaces option it looks like the screenshot currently on the wiki:

![screenshot from 2017-05-11 15-21-26](https://cloud.githubusercontent.com/assets/100427/25974354/867f3a98-365d-11e7-8849-1792a93e7853.png)

However, I feel that it should not require me doing that. That's why I think that the namespace is incorrectly including the node name at all times. Based on the code and `rosnode info` I do not believe these nodes are actually in namespaces:

```
% rosnode info /turtlesim 
--------------------------------------------------------------------------------
Node [/turtlesim]
Publications: 
 * /turtle1/color_sensor [turtlesim/Color]
 * /rosout [rosgraph_msgs/Log]
 * /turtle1/pose [turtlesim/Pose]

Subscriptions: 
 * /turtle1/cmd_vel [geometry_msgs/Twist]

Services: 
 * /clear
 * /turtle1/teleport_relative
 * /turtlesim/set_logger_level
 * /spawn
 * /reset
 * /kill
 * /turtlesim/get_loggers
 * /turtle1/teleport_absolute
 * /turtle1/set_pen


contacting node http://injera:37136/ ...
Pid: 28751
Connections:
 * topic: /rosout
    * to: /rosout
    * direction: outbound
    * transport: TCPROS
 * topic: /turtle1/cmd_vel
    * to: /teleop_turtle (http://injera:46066/)
    * direction: inbound
    * transport: TCPROS
```

**Comment by dirk-thomas** [comment]
Addressed in #8.

**Comment by dirk-thomas** [comment]
Another patch to get rid of the unintentional `color` item: ros-visualization/qt_gui_core#86
